Up2dateService/Up2dateService/Installers/Choco/ChocoInstaller.cs

```diff
@@ -55,31 +62,44 @@ public bool IsPackageInstalled(Package package)
 
         public void Refresh()
         {
-            using (var ps = PowerShell.Create())
+            Process p = new Process();
+            p.StartInfo.FileName = "choco.exe";
+            p.StartInfo.Arguments = "list -l";
+            p.StartInfo.RedirectStandardOutput = true;
+            p.StartInfo.UseShellExecute = false;
+            p.Start();
+            p.WaitForExit();
+            productCodes.Clear();
+            StreamReader standardOutput = p.StandardOutput;
+            while (!standardOutput.EndOfStream)
             {
-                const string psCommand = @"choco list -li";
-                ps.AddScript(psCommand);
-                productCodes.Clear();
-                productCodes.AddRange(ps.Invoke<string>());
+                productCodes.Add(standardOutput.ReadLine());
```

Comment:
After attentive look into the Choco output...
Seems like 1st line shall be skipped.
Also the last line is empty - we don't want it.
And the line above it (last visible one) contains statistics. Also must be excluded.

Reply:
Lines ignored